atom/common/node_bindings.cc

```diff
@@ -220,6 +221,16 @@ void NodeBindings::Initialize() {
   int exec_argc = 0;
   const char** argv = nullptr;
   const char** exec_argv = nullptr;
+
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
+  if (env->HasVar("NODE_OPTIONS")) {
+    std::string options;
+    const char* prog_name = "electron";
+    argv = {&prog_name};
+
+    env->GetVar("NODE_OPTIONS", &options);
```

Comment:
i don't think we need this really

Reply:
Still want to keep it?

Reply:
oh oops we very much need that now
that was from a first iteration when i wasn’t sure we would :see_no_evil: